api/proto/teleport/legacy/types/types.proto

```diff
@@ -5710,6 +5710,17 @@ message DatabaseServiceV1 {
     (gogoproto.nullable) = false,
     (gogoproto.jsontag) = "spec"
   ];
+  // Status is the resource status.
+  DatabaseServiceStatusV1 Status = 3 [
+    (gogoproto.nullable) = false,
+    (gogoproto.jsontag) = "status"
+  ];
+}
+
+// DatabaseServiceStatusV1 is the DatabaseService Status.
+message DatabaseServiceStatusV1 {
```

Comment:
Feels weird to me that HostID is part of metadata but Hostname is status.  Do Hostname change? Is `DatabaseServiceV1` a heartbeat or it has other uses?

Reply:
anything else other than hostname that could be useful?

Reply:
> Do Hostname change?

The hostname can change, if you change the host's hostname (unless you hardcoded it in `teleport.yaml/teleport.nodename`.

> Is DatabaseServiceV1 a heartbeat or it has other uses?

It's only a heartbeat.

> anything else other than hostname that could be useful?

I was thinking about adding the proxied databases in that particular service. Probably in another PR.

Reply:
i was thinking about the cloud identity that the host uses, e.g. AWS caller identity. WDYT? cc @GavinFrazar

Reply:
That sounds helpful as well 👍